browser-extension/common/src/types.ts

```diff
@@ -111,7 +111,7 @@ export interface ScriptObject {
   codeType: ScriptCodeType;
   type: ScriptType;
   value: string;
-  loadTime?: "afterPageLoad" | "beforePageLoad";
+  loadTime?: "afterPageLoad" | "asSoonAsPossible";
```

Comment:
why remove this? might need this for backward compatibility of old rules

```suggestion
  loadTime?: "afterPageLoad" | "beforePageLoad" | "asSoonAsPossible";
```

Reply:
Yes, might be required. Lets keep it